### DISCUSSION TITLE
Pages deploy bot fails

### DISCUSSION THREAD

**Discussion by bryanhanson** [community/community#57193] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

For several hours today my GH pages failed to deploy because of some problem with the bot.  Anyone know what's going on?

https://github.com/bryanhanson/ChemoSpec/actions/runs/5171899620/jobs/9315723454

**Comment by danielalyoshin** [comment]
#32805 deals with the same error you're getting. Perhaps the solution there could be of use?

**Reply by bryanhanson** [reply]
OK will look thanks.

**Reply by danielalyoshin** [reply]
Anytime, I hope it helps!

**Reply by bryanhanson** [reply, marked as answer]
Thank you @danielalyoshin it was the same error; needed to deploy from root rather than `\docs` but I have no idea why, as the action specifies using `\docs`.  Anyway, it's working!

**Reply by danielalyoshin** [reply]
Of course @bryanhanson! Glad I could be of help!
